galaxy/importer/collection.py

```diff
@@ -144,12 +147,53 @@ def _load_contents(self, content_list):
 
             yield content
 
+    def _serialize_contents(self, loader_contents):
+        '''Serialize into json content objects with nested objects'''
+
+        class ContentJSONEncoder(json.JSONEncoder):
+            def default(self, obj):
+                try:
+                    return json.JSONEncoder.default(self, obj)
+                except TypeError:
+                    pass
+
+                try:
+                    return obj.name.lower()
+                except AttributeError:
+                    return str(obj).lower()
```

Comment:
This class isn't going to stay nested in here, is it?

Reply:
(WIP, I know, just asking!)

Reply:
yeah i should move alongside the parent class or into utils